### PR TITLE
[com_fields]List Field Placeholder Parameter is not used

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #14335

Placeholder is not a valid option for a list field.

Apply the PR - check the options tab and you will see a Placeholder field

Set the field type to "list" and the placeholder field is removed

### CODE REVIEW

File: administrator/components/com_fields/models/field.php [hunk 1]
```diff
@@ -957,6 +957,11 @@ protected function preprocessForm(JForm $form, $data, $group = 'content')
 			{
 				$form->setFieldAttribute('default_value', 'description', $key);
 			}
+			//Remove placeholder field on list fields
+			if ($dataObject->type == "list")
+			{
+				$form->removeField('hint','params');
```
Comment (Quy) [inline administrator/components/com_fields/models/field.php]: Add space after comma.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 13179f5236452e19d8c681758cf94b83d68e7f54<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18212">issues.joomla.org/tracker/joomla-cms/18212</a>.</sub>

**Comment by Quy** [comment]
The same can be done with List of Images type.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 302d6ac133a0c3f3d43417afdb734339ac4e691b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18212">issues.joomla.org/tracker/joomla-cms/18212</a>.</sub>

**Comment by brianteeman** [comment]
@quy I am sure it can be done with many ;)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 302d6ac133a0c3f3d43417afdb734339ac4e691b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18212">issues.joomla.org/tracker/joomla-cms/18212</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Comment by zero-24** [comment]
FYI I have just committed a fix for the drone issue: https://github.com/joomla/joomla-cms/commit/c1772f7f395cf84c270c99ad934e40c885284bbd

**Pull request merged**